lib/charms/observability_libs/v0/kubernetes_service_patch.py

```diff
@@ -0,0 +1,280 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
Actually I do not think we care to fix the K8s service at all: this charm does not expose any web api we care to invoke from the outside, and this makes us need —trust. I’d say we just don’t fix this K8s service instead.

Reply:
git-sync has http api for "health" and `/metrics`